gravitee-apim-console-webui/src/services-ngx/api.service.ts

```diff
@@ -375,6 +377,35 @@ export class ApiService {
         .join('&')}`,
     );
   }
+
+  getSubscribers(
+    apiId: string,
+    query?: string,
+    page?: number,
+    size?: number,
+    exclude: ApplicationExcludeFilter[] = [],
+  ): Observable<ApiSubscribersResponse> {
+    const queryParams: string[] = [];
+    if (query) {
+      queryParams.push(`query=${query}`);
+    }
+
+    if (page) {
+      queryParams.push(`page=${page}`);
+    }
+
+    if (size) {
+      queryParams.push(`size=${size}`);
+    }
+
+    if (exclude && exclude.length > 0) {
+      queryParams.push(exclude.map((filter) => `exclude=${filter}`).join('&'));
+    }
+
+    return this.http.get<ApiSubscribersResponse>(
+      `${this.constants.env.baseURL}/apis/${apiId}/subscribers${queryParams ? '?' + queryParams.join('&') : ''}`,
```

Comment:
I think it's more the [rest-api v2](https://github.com/gravitee-io/gravitee-api-management/blob/23bde6df6ec0fb35ca8e9f41ae74456ddd432a8c/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiResource.java#L758) that we would like to use here. wdyt ? 

and add this into ApiV2Service ?

Reply:
It already exists https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/services-ngx/api-v2.service.ts#L183